Raspi/Test/mock_wireless_module.py

```diff
@@ -0,0 +1,170 @@
+        current_time = round(time.time(), 2)
+        total_time = round(current_time - start_time, 2)
+
+        print() # Newline for clarity
+
+        # Wireless Sensor 1 (Middle)
+        if sensorID == 1 or sensorID == None:
+
+            # generate JSON data and publish it over MQTT
+            data = {
+                "CO2": Sensors.CO2(),
+                "temperature": Sensors.temperature(),
+                "humidity": Sensors.humidity(),
+                "accelerometer": Sensors.accelerometer(),
+                "gyroscope": Sensors.gyroscope(),
+                "battery": Sensors.battery(),
+                "time": Sensors.time()}
+
+            JSON_data = json.dumps(data)
+            pub_path = "v3/wireless-sensor/1/data"
+
+            client.publish(pub_path, JSON_data)
+            print(pub_path, "--> ", JSON_data)
+
+        # Wireless Sensor 2 (client, Back)
+        if sensorID == 2 or sensorID == None:
+
+            # generate JSON data and publish it over MQTT
+            data = {
+                "CO2": Sensors.CO2(),
+                "reedSwitch": Sensors.reedSwitch(),
+                "GPS": Sensors.GPS(),
+                "temperature": Sensors.temperature(),
+                "battery": Sensors.battery(),
+                "time": Sensors.time()}
+
+            JSON_data = json.dumps(data)
+            pub_path = "v3/wireless-sensor/2/data"
+
+            client.publish(pub_path, JSON_data)
+            print(pub_path, "--> ", JSON_data)
+
+        # Wireless Sensor 3 (client, Front)
+        if sensorID == 3 or sensorID == None:
+
+            # generate JSON data and publish it over MQTT
+            data = {
+                "steering": Sensors.steering(),
+                "temperature": Sensors.temperature(),
+                "humidity": Sensors.humidity(),
+                "battery": Sensors.battery(),
+                "time": Sensors.time()}
+
+            JSON_data = json.dumps(data)
+            pub_path = "v3/wireless-sensor/3/data"
+
+            client.publish(pub_path, JSON_data)
+            print(pub_path, "--> ", JSON_data)
+
+
+        time.sleep(1/rate)
+        if total_time >= duration:
+            break
+
+def start_publishing(client, args):
+    print("publishing started...")
+
+    send_fake_data(client, args.time, args.rate, args.id)
+
+    print("publishing finished")
+
+def on_connect(client, userdata, flags, rc):
+    if rc == 0:
+        print("Connected Sucessfully! Result code: " + str(rc))
+    else:
+        print("Something went wrong! Result code: " + str(rc))
+
+def on_message(client, userdata, msg):
+    print(msg.topic+" "+str(msg.payload.decode("utf-8")))
+
+def run():
+    args = parser.parse_args()
+    broker_address = args.host
+    client = mqtt.Client()
+
+    client.on_connect = on_connect
+    client.on_message = on_message
+
+    client.connect(broker_address)
+    client.loop_start()
+    start_publishing(client, args)
+    client.loop_stop()
+
+run()
```

Comment:
This causes the `run()` function to execute if you import the python script in another python script.